web/components/navbar.tsx

```diff
@@ -0,0 +1,33 @@
+"use client";
+
+import {
+  RedirectToSignIn,
+  SignedIn,
+  SignedOut,
+  UserButton,
+} from "@clerk/clerk-react";
+import { Navbar, NavbarContent, NavbarItem } from "@nextui-org/react";
+
+export const NavBar = () => {
+  return (
+    <>
+      <SignedIn>
+        <Navbar>
+          <NavbarContent justify="start">
+            <NavbarItem>Your return</NavbarItem>
+            <NavbarItem>Saved</NavbarItem>
+          </NavbarContent>
+          <NavbarContent justify="end">
+            <NavbarItem>Help</NavbarItem>
+            <NavbarItem>
+              <UserButton />
+            </NavbarItem>
+          </NavbarContent>
+        </Navbar>
+      </SignedIn>
+      <SignedOut>
+        <RedirectToSignIn />
```

Comment:
would this make more sense to be in layout.tsx?

Reply:
It has to be in a client component

Reply:
what about putting in the providers component?

Reply:
actually, that would require all routes to be authenticated...
makes sense leaving as-is